tv_apps/smart-system/js/browser_context_menu.js

```diff
@@ -242,10 +252,17 @@
     if (this.scrollable.currentItem) {
       this.scrollable.currentItem.blur();
     }
-    this.element.classList.remove('visible');
-    if (this.app) {
-      this.app.focus();
-    }
+    this.smartDialogElement.close(700, {
+      p1: 0.5,
+      p2: 0,
+      p3: 0,
+      p4: 1
+    }, function() {
+      this.element.classList.remove('visible');
+      if (this.app) {
+        this.app.focus();
```

Comment:
When performing pin to home, this line would take focus back to app-deck after the focus has been switched to home app.
Seems that just performing timeout is not enough. One way I can come out is that postpone calling callbacks of items[1] to here.

[1]https://github.com/sean2449/gaia/blob/Bug_1122421/tv_apps/smart-system/js/browser_context_menu.js#L194